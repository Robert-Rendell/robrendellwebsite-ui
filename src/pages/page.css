.standard-page-margins {
  padding: 10px;
}

.standard-page-styling {
  background-color: #282c34;
  font-size: calc(10px + 2vmin);
  color: white;
}

.centred {
  justify-content: center;
  text-align: center;
}

img {
  -khtml-user-select: none;
  -o-user-select: none;
  -moz-user-select: none;
  -webkit-user-select: none;
  user-select: none;
}

.s3-loaded-image {
  max-width: 400px;
}

.numbered-list-margins {
  padding: 20px;
}

h2 {
  color: white;
}

.black-bg {
  word-wrap: break-word;
  background-color: black;
  height: 100%;
}

.black-bg-auto {
  height: auto;
}

.black-bg a {
  color: limegreen;
}

.black-bg a:hover {
  color: white;
}

.black-bg button {
  background-color: limegreen;
  color: black;
  border-color: black;
}

.black-bg button:hover {
  background-color: limegreen;
  border-color: black;
}

.ol-extra-margins {
  margin-left: 3rem;
  margin-right: 3rem;
  word-wrap: break-word;
}

#mc-edoegsa-page {
  overflow: hidden;
}
