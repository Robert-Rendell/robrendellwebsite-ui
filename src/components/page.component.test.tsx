import React from "react";
import { render } from "@testing-library/react";
import { PageComponent } from "./page.component";

describe("PageComponent", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<PageComponent route="Rob" title="Rob title"/>);
    expect(asFragment()).toMatchSnapshot();
  });
});