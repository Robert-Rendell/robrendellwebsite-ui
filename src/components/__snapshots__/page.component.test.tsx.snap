// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageComponent should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="standard-page-margins standard-page-styling"
  >
    <h1>
      Rob title
    </h1>
    <hr />
    <hr />
  </div>
</DocumentFragment>
`;
